### PR TITLE
Create voting function

### PR DESCRIPTION
**Pull request by nampx-0860**

### CODE REVIEW

File: app/Http/Controllers/User/VoteController.php [hunk 1]
```diff
@@ -35,7 +37,23 @@ public function create()
      */
     public function store(Request $request)
     {
-        //
+        try {
+            $inputPaticipant = $request->only('name', 'email');
+            $inputPaticipant['user_id'] = 1;
```
Comment (framgiaciv3) [inline app/Http/Controllers/User/VoteController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.SuperfluousWhitespace.EndLine</br>**Severity**: error</br>**File**: app/Http/Controllers/User/VoteController.php</br>**Line**: 42</br>Whitespace found at end of line

File: app/Http/Controllers/User/VoteController.php [hunk 2]
```diff
@@ -35,7 +37,23 @@ public function create()
      */
     public function store(Request $request)
     {
-        //
+        try {
+            $inputPaticipant = $request->only('name', 'email');
+            $inputPaticipant['user_id'] = 1; 
+            Participant::create($inputPaticipant);
+
+            $participant_id = Participant::max('id');
```
Comment (framgiaciv3) [inline app/Http/Controllers/User/VoteController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Variables.ValidVariableName.NotCamelCaps</br>**Severity**: error</br>**File**: app/Http/Controllers/User/VoteController.php</br>**Line**: 45</br>Variable "participant_id" is not in valid camel caps format

File: app/Http/Controllers/User/VoteController.php [hunk 3]
```diff
@@ -35,7 +37,23 @@ public function create()
...
+            $inputPaticipantVote['option_id'] = $request->option;
+            $inputPaticipantVote['participant_id'] = $participant_id;
```
Comment (framgiaciv3) [inline app/Http/Controllers/User/VoteController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Variables.ValidVariableName.NotCamelCaps</br>**Severity**: error</br>**File**: app/Http/Controllers/User/VoteController.php</br>**Line**: 47</br>Variable "participant_id" is not in valid camel caps format

File: app/Http/Controllers/User/VoteController.php [hunk 4]
```diff
@@ -57,7 +75,7 @@ public function show($id)
      */
     public function edit($id)
     {
-        //
+        //
```
Comment (framgiaciv3) [inline app/Http/Controllers/User/VoteController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.SuperfluousWhitespace.EndLine</br>**Severity**: error</br>**File**: app/Http/Controllers/User/VoteController.php</br>**Line**: 78</br>Whitespace found at end of line

File: app/Http/Controllers/User/VoteController.php [hunk 5]
```diff
@@ -14,7 +16,7 @@ class VoteController extends Controller
      */
     public function index()
     {
-        //
+
     }
```
Comment (framgiaciv3) [inline app/Http/Controllers/User/VoteController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Methods.FunctionClosingBrace.SpacingBeforeClose</br>**Severity**: error</br>**File**: app/Http/Controllers/User/VoteController.php</br>**Line**: 20</br>Function closing brace must go on the next line following the body; found 1 blank lines before brace

File: app/Http/Controllers/User/VoteController.php [hunk 6]
```diff
@@ -24,7 +26,7 @@ public function index()
      */
     public function create()
     {
-        //
+
     }
```
Comment (framgiaciv3) [inline app/Http/Controllers/User/VoteController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Methods.FunctionClosingBrace.SpacingBeforeClose</br>**Severity**: error</br>**File**: app/Http/Controllers/User/VoteController.php</br>**Line**: 30</br>Function closing brace must go on the next line following the body; found 1 blank lines before brace

File: app/Http/Controllers/User/VoteController.php [hunk 7]
```diff
@@ -46,7 +65,7 @@ public function store(Request $request)
      */
     public function show($id)
     {
-        //
+
     }
```
Comment (framgiaciv3) [inline app/Http/Controllers/User/VoteController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Methods.FunctionClosingBrace.SpacingBeforeClose</br>**Severity**: error</br>**File**: app/Http/Controllers/User/VoteController.php</br>**Line**: 69</br>Function closing brace must go on the next line following the body; found 1 blank lines before brace

File: app/Http/Controllers/User/VoteController.php [hunk 8]
```diff
@@ -57,7 +76,7 @@ public function show($id)
      */
     public function edit($id)
     {
-        //
+
     }
```
Comment (framgiaciv3) [inline app/Http/Controllers/User/VoteController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Methods.FunctionClosingBrace.SpacingBeforeClose</br>**Severity**: error</br>**File**: app/Http/Controllers/User/VoteController.php</br>**Line**: 80</br>Function closing brace must go on the next line following the body; found 1 blank lines before brace

File: app/Http/Controllers/User/VoteController.php [hunk 9]
```diff
@@ -69,7 +88,7 @@ public function edit($id)
      */
     public function update(Request $request, $id)
     {
-        //
+
     }
```
Comment (framgiaciv3) [inline app/Http/Controllers/User/VoteController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Methods.FunctionClosingBrace.SpacingBeforeClose</br>**Severity**: error</br>**File**: app/Http/Controllers/User/VoteController.php</br>**Line**: 92</br>Function closing brace must go on the next line following the body; found 1 blank lines before brace

File: app/Http/Controllers/User/VoteController.php [hunk 10]
```diff
@@ -80,6 +99,6 @@ public function update(Request $request, $id)
      */
     public function destroy($id)
     {
-        //
+
     }
```
Comment (framgiaciv3) [inline app/Http/Controllers/User/VoteController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Methods.FunctionClosingBrace.SpacingBeforeClose</br>**Severity**: error</br>**File**: app/Http/Controllers/User/VoteController.php</br>**Line**: 103</br>Function closing brace must go on the next line following the body; found 1 blank lines before brace

**Comment by nampx-0860** [comment]
Peek 2019-03-27 10-08

**Pull request merged**